### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Adithya2357**
## Pull Request Title
*Improved README Documentation for Metasploit Framework*

### What This Change Does
This pull request improves the clarity and organization of the Metasploit Framework's README documentation. It restructures content into distinct categories, updates installation instructions, enhances usage guidance, and provides a detailed contributing section. The objective is to make the documentation more user-friendly and accessible, helping both new users and contributors find the information they need quickly.

### Issue Reference
This change does not fix a specific bug but aims to enhance the overall user experience. There is no associated GitHub issue number for this documentation update.

### Verification

To ensure the changes are correct and effective, please:

- [ ] Review the updated README content for accuracy and coherence.
- [ ] Verify all links in the README work correctly, pointing to the intended resources.

### Additional Notes
No specific hardware or complex testing environments are required for this documentation change. The primary verification involves ensuring clarity and correctness of the provided information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,52 +1,46 @@
...
 
-New bugs and feature requests should be directed to:
-  https://r-7.co/MSF-BUGv1
+## Development Environment
+To set up a development environment, visit the [Development Setup Guide](https://docs.metasploit.com/docs/development/get-started/setting-up-a-metasploit-development-environment.html).
 
-API documentation for writing modules can be found at:
-  https://docs.metasploit.com/api/
+## Bug and Feature Requests
+Submit bugs and feature requests via the [GitHub Issues](https://github.com/rapid7/metasploit-framework/issues) tracker. New submissions can be made through the [MSF-BUGv1 form](https://r-7.co/MSF-BUGv1).
```
Comment (adfoster-r7) [inline README.md:16]:
```suggestion
Submit bugs and feature requests via the [GitHub Issues](https://github.com/rapid7/metasploit-framework/issues) tracker. New submissions can be made through the [MSF-BUGv1 form](https://github.com/rapid7/metasploit-framework/issues/new/choose).
```

File: README.md [hunk 2]
```diff
@@ -1,52 +1,46 @@
...
 
-Contributing
---
-See the [Dev Environment Setup][devenv] guide on GitHub, which will
-walk you through the whole process from installing all the
-dependencies, to cloning the repository, and finally to submitting a
-pull request. For slightly more information, see
-[Contributing](https://github.com/rapid7/metasploit-framework/blob/master/CONTRIBUTING.md).
+### Recommended Installation
+Use the free installer, which includes all dependencies and provides a simple setup process. For a manual setup, consult the [Dev Environment Setup](https://docs.metasploit.com/docs/development/get-started/setting-up-a-metasploit-development-environment.html) guide.
```
Comment (adfoster-r7) [inline README.md:29]:
```suggestion
We recommend installation with the [official Metasploit installers](https://docs.metasploit.com/docs/using-metasploit/getting-started/nightly-installers.html#installing-metasploit-on-linux--macos) on Linux or macOS. Metasploit is also pre-installed with Kali.

For a manual setup, consult the [Dev Environment Setup](https://docs.metasploit.com/docs/development/get-started/setting-up-a-metasploit-development-environment.html) guide.
```

File: README.md [hunk 3]
```diff
@@ -1,52 +1,46 @@
...
 
+## Using Metasploit
 
-[devenv]: https://docs.metasploit.com/docs/development/get-started/setting-up-a-metasploit-development-environment.html "Metasploit Development Environment Setup"
-[unleashed]: https://www.offensive-security.com/metasploit-unleashed/ "Metasploit Unleashed"
+To get started with Metasploit:
 
+1. **Start `msfconsole`:** This is the primary interface for interacting with Metasploit.
+2. **Explore Resources:** 
+   - Read [Metasploit Unleashed](https://www.offensive-security.com/metasploit-unleashed/) for an in-depth guide.
```
Comment (adfoster-r7) [inline README.md:37]:
```suggestion
```

**Comment by bwatters-r7** [comment]
@Adithya2357 Any response to the suggestions?

**Comment by Adithya2357** [comment]
@bwatters-r7 i think these changes are well enough and @adfoster-r7 did it way more clear

**Review by smcintyre-r7 — APPROVED** [review]

**Comment by smcintyre-r7** [comment]
# Release Notes
This improves the clarity and organization of the Metasploit Framework's README documentation. It restructures content into distinct categories, updates installation instructions, enhances usage guidance, and provides a detailed contributing section.

**Pull request merged**